core/commonTest/src/kotlinx/serialization/features/ThirdPartyGenericsTest.kt

```diff
@@ -0,0 +1,70 @@
+package kotlinx.serialization.features
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.modules.SerializersModule
```

Comment:
We usually prefer star imports in the code style

Reply:
You want this to be a `kotlinx.serialization.*` import? So if there is more than one import from a package, we use '*'? Or one level lower, e.g. `kotlinx.serialization.encoding.*`? Perhaps good to indicate the threshold in CONTRIBUTING.md? 

Out of interest, why this deviation from the ktlint ruleset (https://ktlint.github.io/#rule-import)?

Reply:
We follow official style guide (https://kotlinlang.org/docs/coding-conventions.html) which does not explicitly endorse or prohibit star imports. However, they're proven to be useful because one usually want to import extensions or dependent classes alongside the class itself; and it also looks very neat when library fits in 3-4 packages. For example, Ktor style guide explicitly recommends them for this reason: https://ktor.io/docs/code-style.html#with-star-imports